saml/serviceProviders/aws/provider.go

```diff
@@ -76,9 +76,10 @@ func (a AwsServiceProvider) GetCredentials(saml string, desiredRole string) *sts
 	}
 
 	samlInput := &sts.AssumeRoleWithSAMLInput{
-		PrincipalArn:  aws.String(role.Principal),
-		RoleArn:       aws.String(role.ARN),
-		SAMLAssertion: aws.String(saml),
+		DurationSeconds: aws.Int64(int64(durationInMinutes) * 60),
```

Comment:
Double-check: this will just work with any config files as well? https://github.com/Brightspace/bmx#configuration-parameters

If so, could the README be updated as well?

Reply:
Yes, I'll update the readme